base_classes/NXbeam.nxdl.xml

```diff
@@ -370,16 +370,12 @@
             Direction (and location) for the beam. The location of the beam can be given by
             any point which it passes through as its offset attribute.
         </doc>
-        <field name="DIRECTION" nameType="any" units="NX_TRANSFORMATION" type="NX_NUMBER">
+        <field name="BEAMdirection" nameType="partial" units="NX_UNITLESS" type="NX_NUMBER">
```

Comment:
[This concept will not inherit from this one](https://github.com/nexusformat/definitions/blob/8b111da0e5db9ed5ffe02cce5a7483420fc19c2c/base_classes/NXtransformations.nxdl.xml#L120)

but it defines attributes as if it would inherit from the concept that is behind the above-mentioned permalink.

It is stated here that the concepts ("BEAMdirection") transformation_type attribute should be omitted, which is happening here I understand this but even if it gets omitted shouldnt then BEAMdirection inherit from /NXtransformations/AXISNAME ?